### DISCUSSION TITLE
Add Tables to Insights

### DISCUSSION THREAD

**Discussion by Emsu** [directus/directus#16814] [Feature Requests]
For Insights it would be great to be able to use several of the other Layout options for data such as Tables or Maps.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
